Frontend/AdminServer/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Czy w tym PR było robione coś we frontencdzie?

Reply:
nie mam zielonego pojecia, ale visual  i to dorzuca co chwile. Mozna olac. Nie wiem czy ten plik wplynie na projekt